buildtest/cli/buildspec.py

```diff
@@ -1161,37 +1161,39 @@ def buildspec_validate(
     console.print("[green]All buildspecs passed validation!!!")
 
 
-def summarize_buildspec_cache(pager, configuration):
+def summarize_buildspec_cache(pager, configuration, color=None):
     """entry point for ``buildtest buildspec summary``
 
     Args:
         configuration (buildtest.config.SiteConfiguration): instance of type SiteConfiguration
-        pager (bool): Bolean control output of summary with paging
+        pager (bool): Boolean control output of summary with paging
+        color (str, optional): An instance of str, color that the summary should be printed in
     """
     if pager:
         with console.pager():
             summary_print(configuration)
             return
-    summary_print(configuration)
+    summary_print(configuration, color)
 
 
-def summary_print(configuration):
+def summary_print(configuration, color=None):
     """Prints summary of buildspec cache which is run via command ``buildtest buildspec summary``
 
     Args:
         configuration (buildtest.config.SiteConfiguration): instance of type SiteConfiguration
+        color (str, optional): An instance of str, color that the summary should be printed in
     """
     cache = BuildspecCache(configuration=configuration)
+    consoleColor = checkColor(color)
     msg = f"""
-    [yellow]Reading Buildspec Cache File:[/yellow]   [cyan]{BUILDSPEC_CACHE_FILE}[/cyan] 
-    [yellow]Total Valid Buildspecs:[/yellow]         [cyan]{len(cache.get_valid_buildspecs())}[/cyan] 
-    [yellow]Total Invalid Buildspecs:[/yellow]       [cyan]{len(cache.get_invalid_buildspecs())}[/cyan] 
-    [yellow]Total Unique Tags:[/yellow]              [cyan]{len(cache.get_unique_tags())}[/cyan] 
-    [yellow]Total Maintainers:[/yellow]              [cyan]{len(cache.get_maintainers())}[/cyan] 
+    [{consoleColor}]Reading Buildspec Cache File:   {BUILDSPEC_CACHE_FILE} 
+    [{consoleColor}]Total Valid Buildspecs:         {len(cache.get_valid_buildspecs())} 
+    [{consoleColor}]Total Invalid Buildspecs:       {len(cache.get_invalid_buildspecs())} 
+    [{consoleColor}]Total Unique Tags:              {len(cache.get_unique_tags())} 
+    [{consoleColor}]Total Maintainers:              {len(cache.get_maintainers())} 
```

Comment:
right now we didnt color output of text but only table rows i think to be consistent we shouldn't color these. What do you think? Should we revert back or pursue this change. Note that if we decide to color all text we should be consistent though i must admit this may not be desirable for all text.

Reply:
@shahzebsiddiqui Right, let's not color the text since we have only colored the tables all over. I will revert back the changes.